Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1559,7 +1559,7 @@ void WebPageProxy::didAttachToRunningProcess()
         videoPresentationManager->setMockVideoPresentationModeEnabled(m_mockVideoPresentationModeEnabled);
 #endif
 
-#if ENABLE(APPLE_PAY)
+#if ENABLE(APPLE_PAY) && !ENABLE(APPLE_PAY_REMOTE_UI)
```

Comment:
LGTM, we probably want a better name for this now, since APPLE_PAY_REMOTE_UI is not too telling...

Reply:
... out of scope for this patch, of course